ports/v-hacd/portfile.cmake

```
@@ -0,0 +1,44 @@
+vcpkg_check_linkage(ONLY_STATIC_LIBRARY)

```

```suggestion
vcpkg_fail_port_install(ON_ARCH "arm")

vcpkg_check_linkage(ONLY_STATIC_LIBRARY)

```
We still need to add this here.